callisto/delivery/views.py

```diff
@@ -111,6 +111,7 @@ def submit_to_school(request, report_id, form_template_name="submit_to_school.ht
             except Exception:
                 # report was sent even if confirmation email fails, so don't show an error if so
                 logger.exception("couldn't send confirmation to user on submission")
+                context.update({'form': form, 'email_confirmation_error': True})
```

Comment:
Agreed. I felt weird about changing the views.py so this is better. My only concern with checking if the report was actually sent is that you will most likely be sending emails async at some point in time and the delay may not be consistent - so tracking emails are actually gone doesn't seem to match a potential real world scenario.